api/v1alpha1/oadp_types.go

```diff
@@ -160,6 +160,12 @@ type NodeAgentConfig struct {
 	// +kubebuilder:validation:Enum=restic;kopia
 	// +kubebuilder:validation:Required
 	UploaderType string `json:"uploaderType"`
+
+	// UnsupportedNodeAgentArgs accepts a string of server args and their values to be supplied to nodeagent daemon set
+	// each pair of server arg and value is comma separated if there are multiple args to be supplied
```

Comment:
would add where user can get possible values (`velero help node-agent server`, being the same version as used by OADP) and an example

Reply:
Unsure about this, we do not ship the velero CLI (its part of the velero cotnainer itself, but we do not do velero CLI releases)

Reply:
So a customer could easily run help from the velero container.   I would rather not be explicit about the help here for customers.  The hope is customers only use this when consulting with our support and eng when needed.

Reply:
usage examples can be part of docs

Reply:
ok, resolving